devfiles/quarkus-commandmode/devfile.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: quarkus-commandmode-
+projects:
+  -
+    name: quarkus-quickstarts
+    source:
+      type: git
+      location: "https://github.com/quarkusio/quarkus-quickstarts.git"
+      sparseCheckoutDir: getting-started-command-mode
+components:
+  -
+    type: chePlugin
+    id: redhat/quarkus-java11/latest
+  -
+    type: dockerimage
+    alias: centos-quarkus-maven
+    image: quay.io/eclipse/che-quarkus:nightly
+    env:
+      - name: JAVA_OPTS
+        value: "-XX:MaxRAMPercentage=50.0 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90
+          -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
+          -Duser.home=/home/user"
+      - name: MAVEN_OPTS
+        value: $(JAVA_OPTS)  
+    memoryLimit: 4927M
+    mountSources: true
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+
+  - type: dockerimage
```

Comment:
do we need this image ? we can't use centos-quarkus-maven to read the command arguments ?

Reply:
this image is nice to test the command without any other dependencies: without maven, etc ...

Reply:
yes but here it looks overkill to just read an input value, no ?

Reply:
yes and no, the devfile could be used as template and it may be good for other cli project to test with the minimum. Also showcasing Che capabilities.